[2022-04-13 22:36:10,914] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:31+00:00 [queued]>
[2022-04-13 22:36:10,942] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:31+00:00 [queued]>
[2022-04-13 22:36:10,944] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:10,945] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:36:10,946] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:11,001] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-02 06:00:31+00:00
[2022-04-13 22:36:11,008] {standard_task_runner.py:52} INFO - Started process 11324 to run task
[2022-04-13 22:36:11,021] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:31+00:00', '--job-id', '1954', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmphi3nesk9', '--error-file', '/tmp/tmpg4cexts0']
[2022-04-13 22:36:11,062] {standard_task_runner.py:77} INFO - Job 1954: Subtask ingest
[2022-04-13 22:36:11,170] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:31+00:00 [running]> on host 13db72185839
[2022-04-13 22:36:11,286] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:31+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:31+00:00
[2022-04-13 22:36:11,289] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:36:11,291] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:36:11,303] {subprocess.py:85} INFO - Output:
[2022-04-13 22:36:11,311] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:36:11,311] {subprocess.py:89} INFO - dags
[2022-04-13 22:36:11,313] {subprocess.py:89} INFO - logs
[2022-04-13 22:36:11,313] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:36:11,314] {subprocess.py:89} INFO - output_2022-03.csv
[2022-04-13 22:36:11,315] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:36:11,315] {subprocess.py:89} INFO - plugins
[2022-04-13 22:36:11,316] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:36:11,316] {subprocess.py:89} INFO - scripts
[2022-04-13 22:36:11,317] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:36:11,318] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:36:11,368] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060031, start_date=20220413T223610, end_date=20220413T223611
[2022-04-13 22:36:11,413] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:36:11,457] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
